commcare_connect/opportunity/models.py

```diff
@@ -3,20 +3,31 @@
 from decimal import Decimal
 from uuid import uuid4
 
-from django.conf import settings
 from django.core.validators import MinValueValidator
 from django.db import models
 from django.db.models import Count, F, Max, Q, Sum
 from django.utils.dateparse import parse_datetime
 from django.utils.functional import cached_property
 from django.utils.timezone import now
 from django.utils.translation import gettext
+from oauth2_provider import settings
 
 from commcare_connect.organization.models import Organization
 from commcare_connect.users.models import User
 from commcare_connect.utils.db import BaseModel, slugify_uniquely
 
 
+class HQServer(models.Model):
```

Comment:
This doesn't feel like it is an opportunity based model (and this file is already massive, so wouldn't hurt to break out anyway)

Reply:
https://github.com/dimagi/commcare-connect/pull/666/commits/e8c25ff58ed227d01292032b64b7e0c375d77e1c